lib/steep/server/interaction_worker.rb

```diff
@@ -160,7 +160,9 @@ def response_to_completion(path:, line:, column:, trigger:)
       def format_completion_item(item)
         range = LanguageServer::Protocol::Interface::Range.new(
           start: LanguageServer::Protocol::Interface::Position.new(
-            line: item.range.start.line-1,
+            line: item.range.start.line
```

Comment:
I think we need this `-1` because the line numbers in LSP is zero origin where its one origin in the rest of the world.

Reply:
Oh! Totally didn't mean to remove that! Will add back